doc/source/examples/04_applied/03_ahmed_body_fluent.mystnb

```diff
@@ -0,0 +1,360 @@
+for face in enclosure.faces:
+    if face.surface_type.name == "SURFACETYPE_PLANE":
+        plane_surface.append(face)
+    elif face.surface_type.name == "SURFACETYPE_CYLINDER":
+        cylindrical_surface.append(face)
+
+wall_mount = []
+
+# Identify faces associated with mounting
+for cyl_face in cylindrical_surface:
+    if cyl_face.point(0, 0.5)[1] < 0.050:
+        wall_mount.append(cyl_face)
+
+# Identify faces associated enclosure extremes
+outlet_face, inlet_face = face_identifier(faces=plane_surface, axis="z")
+symmetry_face, symmetry_x_pos = face_identifier(faces=plane_surface, axis="x")
+ground, top = face_identifier(faces=plane_surface, axis="y")
+
+
+# Create Named selection
+design.create_named_selection("wall_mount", faces=wall_mount)
+design.create_named_selection("inlet", faces=[inlet_face])
+design.create_named_selection("outlet", faces=[outlet_face])
+design.create_named_selection("wall_ground", faces=[ground])
+design.create_named_selection("symmetry_top", faces=[top])
+design.create_named_selection("symmetry_center_plane", faces=[symmetry_face])
+design.create_named_selection("symmetry_x_pos", faces=[symmetry_x_pos])
+design.create_named_selection(
+    "ahmed_body_20_0degree_boi_half-boi", faces=boi_body.faces
+)
+
+body_planar_surface = list(
+    set(plane_surface)
+    - set([outlet_face, inlet_face, symmetry_face, symmetry_x_pos, ground, top])
+)
+body_circular_surface = list(set(cylindrical_surface) - set(wall_mount))
+
+rear_surface, front_surface = face_identifier(faces=body_planar_surface, axis="z")
+bottom_surface, top_surface = face_identifier(faces=body_planar_surface, axis="y")
+side_suface, symmetry_surface = face_identifier(faces=body_planar_surface, axis="x")
+
+body_circular_surface.append(front_surface)
+
+design.create_named_selection("wall_ahmed_body_front", faces=body_circular_surface)
+design.create_named_selection(
+    "wall_ahmed_body_main",
+    faces=[symmetry_surface, bottom_surface, top_surface],
+)
+
+# Identify face which intends the slant angle of 20 degrees
+for face in body_planar_surface:
+    if round(math.degrees(math.acos(abs(face.normal().y)))) == 20:
+        hypo_face = face
+design.create_named_selection(
+    "wall_ahmed_body_rear", faces=[hypo_face, rear_surface]
+)
+```
+
+### 6. Export model as a PMDB file
+
+Export the geometry into a Fluent-compatible format. The following code exports the geometry into a PMDB file, which retains the named selections.
+
+```{code-cell} ipython3
+
+# Save the design
+file = design.export_to_pmdb()
+print(f"Design saved to {file}")
+```
+You can import the exported PMDB file into Fluent to set up the mesh and perform the simulation.
+For an example of how to set up the mesh and boundary conditions in Fluent, see the [Ahmed Body External Aerodynamics Simulation](https://examples.fluent.docs.pyansys.com/version/dev/examples/00-released_examples/00-ahmed_body_workflow.html) example in the Fluent documentation.
```

Comment:
@RobPasMue I see the Fluent simulation examples aren't using sentence case headings.

Reply:
Yeah... we should definitely get on top of those - @PipKat if you had the time, that would be awesome but I know that you might have other tasks so don't worry if you can't. I can communicate this to @MaxJPRey -- we should do a full editorial review of their examples in https://github.com/ansys/pyfluent-examples

@agvarghe - could you communicate this to the Fluent team in charge of the examples? I'm guessing its @abhishekchitwar mainly.